src/LLMProviders/chatModelManager.ts

```diff
@@ -73,7 +72,7 @@ export default class ChatModelManager {
         openAIProxyBaseUrl: params.openAIProxyBaseUrl,
       },
       [ModelProviders.ANTHROPIC]: {
-        anthropicApiUrl: `http://localhost:${PROXY_SERVER_PORT}`,
+        anthropicApiUrl: "https://api.anthropic.com",
```

Comment:
I think you can remove this line? ChatAnthropic should default to this.

Reply:
I removed and tested and it works correctly, thanks!